.github/workflows/build.yml

```diff
@@ -64,6 +64,42 @@ jobs:
           activate-environment: flexflow
           environment-file: conda/environment.yml
           auto-activate-base: false
+        
```

Comment:
it's ok to have the code here for now, but we will likely need to move it to another workflow (or a separate job) before merging the PR